api/lib/opentelemetry/trace/trace_flags.rb

```diff
@@ -25,6 +25,10 @@ def from_byte(flags)
         end
       end
 
+      def to_s
```

Comment:
Add a test?

Reply:
so I moved this into the trace parent, as the trace parent defines the flags it can support due to it's version.  It also says that we can only send flags we know about re: version.